pubsub/azuresb/azuresb.go

```diff
@@ -269,6 +269,18 @@ func (t *topic) SendBatch(ctx context.Context, dms []*driver.Message) error {
 	for k, v := range dm.Metadata {
 		sbms.Set(k, v)
 	}
+	if dm.BeforeSend != nil {
```

Comment:
Missing "As" doc comment at top of file?

Reply:
Done.